packages/build/src/compile-exec.ts

```diff
@@ -51,23 +52,24 @@ const compileExec = async(
   execInput: string,
   outputDir: string,
   platform: string,
+  signable: boolean,
   analyticsConfig: string,
   segmentKey: string): Promise<string> => {
-  // Nexe has a huge problem figuring out dependencies in this project,
-  // especially with all the lerna symlinking, so we use Parcel to bundle
-  // up everything into a single JS under cli-repl/dist/mongosh.js
-  // that Nexe can make an executable of. This JS also takes care of the
-  // analytics config file being written.
+  // We use Parcel to bundle up everything into a single JS under
+  // cli-repl/dist/mongosh.js that the executable generator can use as input.
+  // This JS also takes care of the analytics config file being written.
   await generateInput(input, execInput, analyticsConfig, segmentKey);
 
   const executable = executablePath(outputDir, platform);
   console.log('mongosh: creating binary:', executable);
 
-  if (platform === Platform.MacOs) {
-    const { exec } = require('pkg');
-    await new UnsignableCompiler(input, executable, platform)
-      .compile(exec);
+  if (signable) {
```

Comment:
will we ever use the non-signable one in the ci?

Reply:
I don’t think so, no … happy to remove `pkg` support altogether if that makes sense to you